applications/serial_lte_modem/src/slm_at_commands.c

```diff
@@ -217,9 +217,9 @@ static int handle_at_slmuart(enum at_cmd_type type)
 		case 460800:
 		case 921600:
 		case 1000000:
-			k_delayed_work_init(&slm_work.work, set_uart_wk);
+			k_work_init_delayable(&slm_work.work, set_uart_wk);
 			slm_work.data = baudrate;
-			k_delayed_work_submit(&slm_work.work, K_MSEC(50));
+			k_work_reschedule(&slm_work.work, K_MSEC(50));
```

Comment:
Could be a `schedule`

Reply:
I assumed here that it's better to reset the delay value in case the command is issued twice from the host, `schedule` wouldn't do that.

Reply:
Right, yes. If it isn't being initialized here, I agree